src/kibana/plugins/dashboard/index.html

```diff
@@ -27,21 +29,21 @@
 
     </form>
 
-    <div class="button-group">
+    <div class="button-group" role="toolbar">
       <kbn-tooltip text="New Dashboard" placement="bottom" append-to-body="1">
-        <button ng-click="newDashboard()"><i class="fa fa-file-new-o"></i></button>
+        <button ng-click="newDashboard()" aria-label="New Dashboard"><i aria-hidden="true" class="fa fa-file-new-o"></i></button>
       </kbn-tooltip>
       <kbn-tooltip text="Save Dashboard" placement="bottom" append-to-body="1">
-        <button ng-click="configTemplate.toggle('save')"><i class="fa fa-save"></i></button>
+        <button aria-label="Save Dashboard" aria-haspopup="true" aria-expanded="{{ toggleSaveDashboard ? 'true' : 'false' }}" ng-click="configTemplate.toggle('save'); toggleSaveDashboard = !toggleSaveDashboard"><i aria-hidden="true" class="fa fa-save"></i></button>
```

Comment:
It's very verbose, and without the ternary operator you get the exact same values.